bundles/core/org.eclipse.smarthome.core.audio/src/main/java/org/eclipse/smarthome/core/audio/AudioManager.java

```diff
@@ -42,28 +46,69 @@
      * @param audioStream The audio stream to play
      * @param sinkId The id of the audio sink to use or null
      */
-    void play(AudioStream audioStream, String sinkId);
+    void play(AudioStream audioStream, @Nullable String sinkId);
+
+    /**
+     * Plays the passed audio stream on the given sink.
+     *
+     * @param audioStream The audio stream to play
+     * @param sinkId The id of the audio sink to use or null
+     * @param volume The volume to be used or null
+     */
+    void play(AudioStream audioStream, @Nullable String sinkId, @Nullable PercentType volume);
 
     /**
      * Plays an audio file from the "sounds" folder using the default audio sink.
      *
+     * @param fileName The file from the "sounds" folder
      * @throws AudioException in case the file does not exist or cannot be opened
      */
     void playFile(String fileName) throws AudioException;
 
+    /**
+     * Plays an audio file with the given volume from the "sounds" folder using the default audio sink.
+     *
+     * @param fileName The file from the "sounds" folder
+     * @param volume The volume to be used or null
```

Comment:
Here and on some other functions I miss a statement what will be done if the parameter is `null`.
On some places you already added it e.g.

```text
@param url The url to stream from or null if streaming should be stopped
```

but e.g. on the volume it is missing. Should it be "or null if the default notification volume should be used"?